index.js

```diff
@@ -2,6 +2,97 @@ const hamburgerButton = document.querySelector('.hamburger');
 const closeButton = document.querySelector('.close-button');
 const navItems = document.querySelector('.nav_list');
 const myName = document.querySelector('.logo');
+const body = document.getElementById('body');
+const popupWindow = document.querySelector('.popup_card');
+const header = document.getElementById('header');
+const projectData = [
+  {
+    name: 'Multi-Post Stories',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    featuredImage: [
+      './assets/multi_post_image_desktop.svg',
+      './assets/multi_post_image_desktop.svg',
```

Comment:
Hi @joskalenda, thank you so much for your review, however, I think you added someone else's  review on my project because I don't have  the template you are talking about

Reply:
Hey, @jmonto55, thanks for contacting me I saw your message late today since I went off line, but I apologize for this inconvenient.